plugins/inputs/fritzbox/README.md

```diff
@@ -0,0 +1,179 @@
+# Fritzbox Input Plugin
+
+This input plugin gathers status from [AVM][1] devices (routers, repeaters,
+...). It uses the device's [TR-064][2] interfaces to retrieve the status.
+
+[1]: https://avm.de/
```

Comment:
```suggestion
[1]: https://en.avm.de/
```
English website is preferable. Is there also an English page about TR-064?

Reply:
Good point. Will change.

Reply:
Sadly no English page is available for the developer page.

Reply:
Done via fd007fccfb4d6a2e9825ed61f5b702dc2346c2ca